src/sdk/sql_cluster_router.cc

```diff
@@ -1837,7 +1837,13 @@ std::shared_ptr<hybridse::sdk::ResultSet> SQLClusterRouter::HandleSQLCmd(const h
                 *status = {StatusCode::kCmdError, "Failed to parse job id: " + cmd_node->GetArgs()[0]};
                 return {};
             }
-            return this->GetJobResultSet(job_id);
+
+            auto rs = this->GetJobResultSet(job_id, status);
+            if (status->IsOK()) {
```

Comment:
No need to check if status is ok? If status is not ok, GetJobResultSet will return `{}`
Or you can prepend the status msg in `else`, to let us know where we got the error, in show job, select into or others

Reply:
Okay.